src/cli/commands/force/lightning/local/setup.ts

```diff
@@ -53,12 +45,20 @@ export class Setup extends SfdxCommand implements HasRequirements {
     ];
 
     public async run(): Promise<any> {
-        await this.init(); // ensure init first
+        try {
+            await this.init(); // ensure init first
+        } catch (error) {
+            if (error instanceof SfdxError) {
+                const sfdxError = error as SfdxError;
+                sfdxError.actions = this.examples;
+                throw sfdxError;
+            }
+            throw error;
+        }
```

Comment:
This is a pattern I'm seeing here and also in the downstream libraries(as I'm using the tarball to go ahead and work with locally) which I am not that fond of. So "platform" and "api level" checks are foundational to most of our commands. When something goes wrong we want to show "examples" of command. The problem is that it's not always `Setup`'s examples but also respective command's example. So, when the `validate` functions run thanks to `init()` when it fails I catch the error and if it is a type of `SfdxError` I assume that it's a validation error. Then I set the current command's examples as `actions` and rethrow the error.